### ISSUE TITLE
when the theme-related documents can be finished

### ISSUE THREAD

**Issue by AngleMi** [vuepress/vuepress-next#32]
### Question
I would like to ask when the theme-related documents can be finished. In vuepress1.0, I can modify palette.styl to justify default style, but in vuepress-next, the palette file is removed,  how to modify default style ecpect modify index.styl?

**Comment by meteorlxy** [comment]
Please stop opening these kind of useless issues.

> but in vuepress-next, the palette file is removed

Why do you think it is removed?

**Comment by AngleMi** [comment]
Maybe my description is not accurate enough. Modifying the style of the default variable in palette.styl does not take effect. See the source code is in _variables.styl, so I don’t know how to overwrite the style of the default variable. And why the question of when the theme-related documents can be finished is an unseless issues? In other words, does anyone want to understand that the vuepress-next plan is useless? The issue raised for the first time was closed without any reply. I didn’t understand it very well, so I raised the issue again, I’m sorry that aroused your disgust.

**Comment by AngleMi** [comment]
> Please stop opening these kind of useless issues.
> 
> > but in vuepress-next, the palette file is removed
> 
> Why do you think it is removed?
Maybe my description is not accurate enough. Modifying the style of the default variable in palette.styl does not take effect. See the source code is in _variables.styl, so I don’t know how to overwrite the style of the default variable. And why the question of when the theme-related documents can be finished is an unseless issues? In other words, does anyone want to understand that the vuepress-next plan is useless? The issue raised for the first time was closed without any reply. I didn’t understand it very well, so I raised the issue again, I’m sorry that disturb you.
